captures/tft.py

```diff
@@ -0,0 +1,220 @@
+# Detergent's TFT Bot

```

Comment:
I think you meant for this file to be in the repo root directory, so that it would do a diff and not add a completely new `tft.py` into the `captures` directory?